hazelcast-enterprise/Dockerfile

```diff
@@ -35,32 +39,44 @@ ENV HZ_HOME="${HZ_HOME}" \
     CLASSPATH="" \
     JAVA_OPTS=""
 
+LABEL name="Hazelcast IMDG Enterprise" \
+      maintainer="info@hazelcast.com" \
+      vendor="Hazelcast, Inc." \
+      version="8.1" \
+      release="${HZ_VERSION}" \
+      url="http://www.hazelcast.com" \
+      summary="Hazelcast IMDG Enterprise Image, certified to RHEL 8.1" \
+      description="Starts a standalone Hazelcast server instance to form a cluster based on Kubernetes discovery inside Openshift/Kubernetes"
+
 # Expose port
 EXPOSE 5701
 
 COPY *.sh *.yaml *.jar *.properties ${HZ_HOME}/
+COPY licenses /licenses
 
 # Install
-RUN echo "Installing new APK packages" \
-    && apk add --no-cache openjdk11-jre bash apr curl procps nss \
-    && echo "Installing old OpenSSL 1.0 from Alpine 3.8" \
-    && apk add --no-cache --repository http://dl-cdn.alpinelinux.org/alpine/v3.8/main/ openssl=1.0.2u-r0 \
+RUN mkdir -p "${HZ_HOME}/lib" "$HZ_CP_MOUNT" \
```

Comment:
Do we need this `HZ_CP_MOUNT`? I'd keep the directories structure similar to `hazelcast-oss`.

Reply:
👍